scanner/library.go

```diff
@@ -4,12 +4,19 @@ import (
 	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
 	"github.com/aquasecurity/trivy/pkg/purl"
 	"github.com/aquasecurity/trivy/pkg/types"
+	"github.com/samber/lo"
 
 	"github.com/future-architect/vuls/logging"
 	"github.com/future-architect/vuls/models"
 )
 
 func convertLibWithScanner(apps []ftypes.Application) ([]models.LibraryScanner, error) {
+	for i := range apps {
+		apps[i].Libraries = lo.Filter(apps[i].Libraries, func(lib ftypes.Package, index int) bool {
+			return !lib.Dev
```

Comment:
```suggestion
		apps[i].Libraries = lo.Filter(apps[i].Libraries, func(pkg ftypes.Package, _ int) bool {
			return !pkg.Dev
```

Reply:
I think the context here is library scan. Vuls (almost?) uses the word "package" as OS distro's packages, right?
I want to understand which is preferable pkg or lib here.

Reply:
I've reconsidered, and I think it's fine to leave it as `lib`.

Reply:
Since it was ftypes.Package, I thought pkg would be fine, but looking at other parts(https://github.com/future-architect/vuls/blob/e0a076bb9de194d6abe7129dbd8a1caa9c6049ce/scanner/library.go#L23), it seems that `lib` is often used, so I thought lib would be fine.